core/src/main/java/ai/timefold/solver/core/config/util/ConfigUtils.java

```diff
@@ -372,12 +372,19 @@ public static List<Member> getAllMembers(Class<?> baseClass, Class<? extends Ann
                     .filter(field -> field.isAnnotationPresent(annotationClass) && !field.isSynthetic())
                     .sorted(alphabeticMemberComparator);
             var methodStream = Stream.of(clazz.getDeclaredMethods())
-                    .filter(method -> method.isAnnotationPresent(annotationClass) && !method.isSynthetic())
-                    .sorted(alphabeticMemberComparator);
+                    .filter(method -> method.isAnnotationPresent(annotationClass) && !method.isSynthetic());
+
+            for (var implementedInterface : clazz.getInterfaces()) {
+                methodStream = Stream.concat(
+                        methodStream,
+                        Arrays.stream(implementedInterface.getMethods())
+                                .filter(method -> method.isAnnotationPresent(annotationClass) && !method.isSynthetic()));
+            }
```

Comment:
Since we're already using stream concatenation, no need to mix normal iteration with it too. Use `flatMap` to achieve the same result.